### PR TITLE
Update Segment helper to conform to segment spec

### LINKED ISSUE #15: Segment uses Experiement Viewed instead of Experiment Play
Thanks for this great library.

I checked Segment documentation and it uses Experiment Viewed.

https://segment.com/docs/spec/ab-testing/

### PR DESCRIPTION
**Pull request by buob**
- Match `Experiment Viewed` spec per ab-testing docs
  https://segment.com/docs/spec/ab-testing/
- Use past tense verbiage for Experiment win
  https://segment.com/academy/collecting-data/naming-conventions-for-clean-data/
- Update docs to show window.analytics instead of window.segment

Close #15

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -643,13 +643,13 @@ Remove `win` and `play` listeners and stop reporting results to Mixpanel.
 
 ### `segmentHelper`
 
-Sends events to [Segment](https://segment.com). Requires `window.segment` to be set using [Segment's embed snippet](https://segment.com/docs/libraries/analytics.js/quickstart/#step-1-copy-the-snippet).
+Sends events to [Segment](https://segment.com). Requires `window.analytics` to be set using [Segment's embed snippet](https://segment.com/docs/libraries/analytics.js/quickstart/#step-1-copy-the-snippet).
```
Comment (Akurganow) [inline README.md]: You should change other mentions of the `window.segment` and `segment.track`

**Comment by peterpme** [comment]
:wave: any thing I can do to help move this along?

**Comment by wehriam** [comment]
@peterpme just made you a collaborator. Let me know after you've reviewed this and run through the tests, I'll issue a new npm release. 

Unfortunately I'm swamped these days and don't have much time to contribute. Apologies for any delays!

**Comment by peterpme** [comment]
No worries. Thanks a ton @wehriam! I'll help push these along :smile:

**Comment by peterpme** [comment]
Merging this in. Will update `window.analytics` myself

**Pull request merged**